symfony/webpack-encore-bundle/1.0/config/packages/webpack_encore.yaml

```diff
@@ -0,0 +1,4 @@
+webpack_encore:
+    # The path where Encore is building the assets.
+    # This should match Encore.setOutputPath() in webpack.config.js.
+    output_path: '%kernel.public_dir%/build'
```

Comment:
What about using two separate build directories per dev/prod env? It useful when you should testing in both modes in same time.

Reply:
I don't think that it's really common to run dev & prod at the same time in the same environment...